testing/local-binder-k8s-hub/install-jupyterhub-chart

```diff
@@ -10,7 +10,9 @@ import sys
 from subprocess import check_call
 from tempfile import NamedTemporaryFile
 
-from ruamel import yaml
```

Comment:
This was deprecated syntax on how to use ruamel.yaml, and transitioning away from it like done elsewhere solved the issue.